src/Microsoft.Health.Dicom.Core/Extensions/DicomMediatorExtensions.cs

```diff
@@ -146,5 +139,26 @@ public static class DicomMediatorExtensions
             EnsureArg.IsNotNull(mediator, nameof(mediator));
             return mediator.Send(new ChangeFeedLatestRequest(includeMetadata), cancellationToken);
         }
+
+        public static Task<AddCustomTagResponse> AddCustomTagsAsync(
+            this IMediator mediator, IEnumerable<CustomTagEntry> customTags, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNull(mediator, nameof(mediator));
+            return mediator.Send(new AddCustomTagRequest(customTags), cancellationToken);
+        }
+
+        public static Task<GetAllCustomTagsResponse> GetAllCustomTagsAsync(
+            this IMediator mediator, string customTagPath, CancellationToken cancellationToken)
```

Comment:
no need for customTagPath?

Reply:
yup, put the change to remove this in the wrong branch. :(